[2023-11-29T07:20:19.198+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: airflow_stores_pg_etl.check_sub_file_exists manual__2023-11-29T07:19:51.940410+00:00 [queued]>
[2023-11-29T07:20:19.203+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: airflow_stores_pg_etl.check_sub_file_exists manual__2023-11-29T07:19:51.940410+00:00 [queued]>
[2023-11-29T07:20:19.203+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-11-29T07:20:19.204+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-11-29T07:20:19.204+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-11-29T07:20:19.237+0000] {taskinstance.py:1300} INFO - Executing <Task(FileSensor): check_sub_file_exists> on 2023-11-29 07:19:51.940410+00:00
[2023-11-29T07:20:19.242+0000] {standard_task_runner.py:55} INFO - Started process 26990 to run task
[2023-11-29T07:20:19.245+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'airflow_stores_pg_etl', 'check_sub_file_exists', 'manual__2023-11-29T07:19:51.940410+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpum0nu839']
[2023-11-29T07:20:19.246+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask check_sub_file_exists
[2023-11-29T07:20:19.301+0000] {task_command.py:388} INFO - Running <TaskInstance: airflow_stores_pg_etl.check_sub_file_exists manual__2023-11-29T07:19:51.940410+00:00 [running]> on host codespaces-0b6ae8
[2023-11-29T07:20:19.482+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=airflow_stores_pg_etl
AIRFLOW_CTX_TASK_ID=check_sub_file_exists
AIRFLOW_CTX_EXECUTION_DATE=2023-11-29T07:19:51.940410+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-29T07:19:51.940410+00:00
[2023-11-29T07:20:19.485+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-11-29T07:20:19.486+0000] {filesystem.py:64} INFO - Poking for file /workspaces/hands-on-introduction-data-engineering-4395021/dataset_temp/Different_stores_dataset.csv
[2023-11-29T07:20:19.486+0000] {filesystem.py:69} INFO - Found File /workspaces/hands-on-introduction-data-engineering-4395021/dataset_temp/Different_stores_dataset.csv last modified: 20231111182456
[2023-11-29T07:20:19.487+0000] {base.py:228} INFO - Success criteria met. Exiting.
[2023-11-29T07:20:19.491+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=airflow_stores_pg_etl, task_id=check_sub_file_exists, execution_date=20231129T071951, start_date=20231129T072019, end_date=20231129T072019
[2023-11-29T07:20:19.538+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-11-29T07:20:19.547+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
